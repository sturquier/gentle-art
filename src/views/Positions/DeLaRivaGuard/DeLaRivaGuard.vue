<template>
  <h1>De la Riva guard</h1>
</template>
