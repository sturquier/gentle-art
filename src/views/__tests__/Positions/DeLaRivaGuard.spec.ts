import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import DeLaRivaGuard from '../../Positions/DeLaRivaGuard.vue'

describe('DeLaRivaGuard', () => {
  it('renders properly', () => {
    const wrapper = mount(DeLaRivaGuard)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
