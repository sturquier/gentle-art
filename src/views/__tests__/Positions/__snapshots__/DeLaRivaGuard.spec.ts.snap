// Vitest Snapshot v1

exports[`DeLaRivaGuard > renders properly 1`] = `"<h1>De la Riva guard</h1>"`;
